third_party/bazel/rules_jvm_external_6.0.patch

```diff
@@ -0,0 +1,145 @@
+commit ec04b5431181421608c8ce55d1cb53626d51ab3e
```

Comment:
for some reason in bzlmod you cannot reference this patch from the upstream repo, so I copied it here